observatory-dags/observatory/dags/workflows/crossref_fundref_telescope.py

```diff
@@ -33,7 +33,7 @@
 from airflow.api.common.experimental.pool import create_pool
 from airflow.exceptions import AirflowException
 from airflow.models.taskinstance import TaskInstance
-from observatory.platform.telescopes.snapshot_telescope import (
+from observatory.platform.workflows.snapshot_workflow import (
```

Comment:
I still prefer to call it snapshot_telescope, as it is specialised for ingesting data. Would you extend it for another purpose?